docs/rules/no-mixed-requires.md

```diff
@@ -19,6 +19,14 @@ This rule comes with one boolean option called `grouping` which is turned off by
 }
 ```
 
+The second way to configure this rule is with an object.
```

Comment:
I'd put this as the first way to configure the rule and mention the Boolean (original) option is deprecated